pkg/queue/sqs.go

```diff
@@ -448,16 +457,15 @@ func (s *SQS) poll(key string, queueSpec QueueSpec) {
 				queueSpec.name, err)
 			return
 		} else {
-			klog.Fatalf("Unable to get approximate messages not visible in queue %q, %v.",
+			klog.Errorf("Unable to get approximate messages not visible in queue %q, %v.",
 				queueSpec.name, err)
 		}
 	}
 	// klog.Infof("approxMessagesNotVisible=%d", approxMessagesNotVisible)
 
 	if approxMessagesNotVisible > 0 {
 		klog.Infof("%s: approxMessagesNotVisible > 0, not scaling down", queueSpec.name)
-		s.waitForShortPollInterval()
-		return
+		s.queues.updateMessage(key, approxMessages+approxMessagesNotVisible)
```

Comment:
@justjkk is this to solve this https://github.com/practo/k8s-worker-pod-autoscaler/issues/73 ?